scripts/kubecf-build.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+source scripts/include/setup.sh
+
+require_tools bosh git helm jq j2y ruby y2j
+
+if [[ ! "$(git submodule status -- src/cf-deployment)" =~ ^[[:space:]] ]]; then
+    die "git submodule for cf-deployment is uninitialized or not up-to-date"
+fi
+
+HELM_DIR="${TEMP_DIR}/helm"
+
+[ -d "${HELM_DIR}" ] && rm -rf "${HELM_DIR}"
+mkdir "${HELM_DIR}"
+
+cp -a deploy/helm/kubecf/ "${HELM_DIR}"
```

Comment:
```suggestion
cp -a deploy/helm/kubecf/* "${HELM_DIR}"
```
otherwise `make lint` fails with

```
mkdir: cannot create directory ‘/home/dimitris/workspace/suse/kubecf/output/tmp/helm/assets’: Not a directory
```

`make kubecf-build` also fails

Reply:
Seems to be implementation defined behaviour, and GNU and BSD implement it differently. :(

Fixed in 0b1b2e3.